file_handler.c

```diff
@@ -22,23 +26,29 @@ int is_guiding_line(char *line_pointer) {
 
     char *token = strtok(line, " ");
     if (token == NULL) {
-        return 0;
+        return 0; /* ##CR: Wasn't this case covered when we check if the line is empty?
+        I mean, token will be NULL at this point only if the line is empty. But if it's empty,
+        we'll not get to this part of the code anyway */

```

Comment:
no. token will be null if there are no spaces in the line

Reply:
Yes, in "empty" I meant only spaces. In the is_empty function, you also consider lines with only spaces as - empty

Reply:
Forget about it. Now I understand.